om4labs/catalogs/obs_catalog_gfdl.yml

```diff
@@ -36,6 +36,38 @@ sources:
     metadata:
       origin_url: ''
 
+  Argo_MLD_003:
```

Comment:
let's make a note to copy these files into a central location. You can also include a notebook with the preprocessing steps in the `om4labs/data` directory.